dist/js/util.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
```

Comment:
So files in dist folder are created by babel which trasnpiles es6 code to es5. So our main files are in src folder. I am not sure if we should edit the dist folder files at all.